GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -421,10 +421,19 @@ public static string DeleteTagCmd(string tagName)
 
         public static string SubmoduleUpdateCmd(string name)
         {
-            if (string.IsNullOrEmpty(name))
-                return "submodule update --init --recursive";
+            name = name ?? "";
+            return SubmoduleUpdateCommand(name.Trim().QuoteNE());
+        }
+
+        public static string SubmoduleUpdateCmd(IEnumerable<string> submodules)
+        {
+            string submodulesQuoted = String.Join(" ", submodules.Select(s => s.Trim().QuoteNE()));
+            return SubmoduleUpdateCommand(submodulesQuoted);
+        }
 
-            return "submodule update --init --recursive \"" + name.Trim() + "\"";
+        private static string SubmoduleUpdateCommand(string name)
+        {
+            return "submodule update --init --recursive " + name;
```

Comment:
Can we extract these into a separate class - `SubmoduleUpdateCmd : Gitcommand`?

Reply:
What do you want to test here? How _submodule update_ works? What would the benefit be with a test here?

Reply:
If one was to write tests, those would be that the *command* is generated correctly, not that it executes correctly